model/src/main/java/org/cbioportal/model/PatientClinicalData.java

```diff
@@ -0,0 +1,25 @@
+package org.cbioportal.model;
+
+import org.cbioportal.model.summary.ClinicalDataSummary;
+
+public class PatientClinicalData extends ClinicalDataSummary {
+
+    private Patient patient;
+    private ClinicalAttribute clinicalAttribute;
```

Comment:
Do you have the potential for duplicating a lot of ClinicalAttribute data if you return a collection of PatientClinicalData?